api/src/test/resources/testAppDataDir/configuration/jsonkeyvalues/config.json

```diff
@@ -23,7 +23,7 @@
   "report.test.active": "no",
   
   "report.opdconsult.active": "true",
-  "report.opdconsult.diagnosisQuestion.concepts":["95312123-e0c2-466d-b6b1-cb6e990d0d65"],
+  "report.opdconsult.diagnosisQuestion.concepts":["95312123-e0c2-466d-b6b1-cb6e990d0d65", "f5a541ae-28bc-4471-826f-7e089644cd11"],
```

Comment:
In regards to all the concepts that we refer to here:
```json
"report.opdconsult.diagnosisQuestion.concepts":["95312123-e0c2-466d-b6b1-cb6e990d0d65", "f5a541ae-28bc-4471-826f-7e089644cd11"],
"report.opdconsult.diagnoses.conceptSet":"b3ea209f-9979-481b-9c64-b6dd883c75bf",
"report.opdconsult.referredTo.concept":"96408258-000b-424e-af1a-403919332938"
```
Let's have them being setup in our own dataset with meaningful concept enames:
* `95312123-e0c2-466d-b6b1-cb6e990d0d65` → _Patient diagnosis (recorded at the clinic)_
* `f5a541ae-28bc-4471-826f-7e089644cd11` → _Patient diagnosis (recorded at home)_
* `96408258-000b-424e-af1a-403919332938` → _Is the patient being referred to_

Also avoid overriding an existing concept, just create and use a new one.

Don't hesitate to put a comment above each concept in our test dataset so that we know what is what. For example:
```xml
<!-- Diagnoses concept set -->
<concept concept_id="1926" retired="false" datatype_id="4" class_id="10"
     is_set="true" creator="1" date_created="2008-08-18 12:38:58.0"
     version="" uuid="b3ea209f-9979-481b-9c64-b6dd883c75bf"/>
```